lib/help.js

```diff
@@ -26,6 +26,11 @@ class Help {
    * @returns {Command[]}
    */
 
+  /**
```

Comment:
There is already an existing JSDoc for this routine. This should presumably be a modification rather than an addition. (This is true of many of the additions.)